lib/build.js

```diff
@@ -139,7 +139,11 @@ module.exports = {
       this.buildView.link(text, id, callback);
     });
 
-    this.refreshTargets().catch(function(e) {});
+    var _this = this;
+    var onInit = atom.packages.onDidActivateInitialPackages(function() {
+        _this.refreshTargets().catch(function(e) {});
+        onInit.dispose();
+    });
```

Comment:
This uses babel, you should use `=>` instead to bind the context. Then you can use `this` in the function.

Also, for future reference, this is completely isolated from the rest of the changes you've made. Would've been good have this in a separate PR so that it could be merged regardless of the other code's status.